docs/src/modules/components/Link.tsx

```diff
@@ -14,6 +14,8 @@ interface NextLinkComposedProps
   href?: NextLinkProps['href'];
 }
 
+const Anchor = styled('a')({ cursor: 'pointer' });
```

Comment:
This looks wrong, the platform around adds the cursor pointer on link.

Reply:
will remove the `cursor: pointer` but using `styled` so that the consumer can pass `sx` props.